<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wait</class>
 <widget class="QWidget" name="Wait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(176, 224, 230);</string>
  </property>
  <widget class="QLabel" name="ques">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Are you ready ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YES">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>READY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>145</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>206</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>265</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>146</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>205</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
  <widget class="QLabel" name="situ6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>266</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>waiting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
